src/tests/simple_dfs.c

```diff
@@ -48,8 +48,16 @@ main(int argc, char **argv)
 	rc = dfs_init();
```

Comment:
changes in this file are unrelated to the PR. a wrong merge perhaps?

Reply:
oops